src/resources/Resolver.ts

```diff
@@ -31,13 +31,16 @@ export class Resolver {
       FieldName: this.config.field,
     };
 
-    const isJsResolver = !(
+    const isUnitJsResolver =
+      this.config.kind === 'UNIT' && 'code' in this.config;
+
+    const isPipelineJsResolver = !(
       this.config.kind === 'UNIT' ||
       'request' in this.config ||
       'response' in this.config
     );
 
-    if (!isJsResolver) {
+    if (!isUnitJsResolver && !isPipelineJsResolver) {
```

Comment:
It's been a while since I wrote this code and with this change it became hard to wrap my head around it.. 

If I read this correctly, we should now be able to simplify this condition to something like `isVTLResolver` where 
```
const isVTLResolver = 'request' in this.config || 'response' in this.config;
```

Reply:
Thanks.
I added `isVTLResolver` flag.